specs/Transportation/TrafficFlowObserved/doc/spec.md

```diff
@@ -19,6 +19,15 @@ The data model is defined as shown below:
     -   Attribute type: Text or URL
     -   Optional
 
+-   `vehicleType` : Type of vehicle from the point of view of its structural characteristics.
+    -   See definition at [Vehicle](../../Vehicle/doc/spec.md).
+    -   Attribute metadata:
+        -   `subType` : A string value that allows more granular breakdown of the allowed vehicleType. Eg if the vehicleType is
```

Comment:
do you think that metadata is a good solution for the subType?

Why don't we define a new attribute `vehicleSubType`

Reply:
@jmcanterafonseca , is there any way we can discuss this or resolve this directly, communicating via message board and then waiting for a reply is not working. I need to get this resolved as soon as I can.

Defining vehicleSubType as a new attribute may work, but would we need to define the acceptable values for this attribute, as happens with the vehicleType attribute?

Reply:
Hi @jmcanterafonseca , I have added in the vehicleSubType as a string attribute, and checked the commits against Travis CI, it fails one test, but the failure is a dead link in a different model, and so is nothing to do with my commits. 
thanks for your help

Reply:
@jmcanterafonseca sorry for my noobiness, not used to working with merges etc. Hopefully I have done it correctly now, I think I have added the changes to the correct branch by rebasing?